prov/gni/src/gnix_mr.c

```diff
@@ -146,6 +148,52 @@ static inline int __mr_cache_key_comp(
 }
 
 /**
+ * Pushes an entry into the LRU cache. No limits are maintained here as
+ *   the hard_stale_limit attr value will directly limit the lru size
+ *
+ * @param[in] cache  a memory registration cache object
+ * @param[in] entry  a memory registration cache entry
+ *
+ * @return           FI_SUCCESS, always
+ */
+static inline int __mr_cache_lru_push(
```

Comment:
I can change this from push/pop to enqueue/dequeue

-- Jim




On 6/2/15, 11:26 AM, "ztiffany" <notifications@github.com> wrote:

>In 
>prov/gni/src/gnix_mr.c
><https://github.com/ofi-cray/libfabric-cray/pull/202#discussion_r31541608>
>:
>> @@ -146,6 +148,52 @@ static inline int __mr_cache_key_comp(
>>  }
>>  
>>  /**
>> + * Pushes an entry into the LRU cache. No limits are maintained here as
>> + *   the hard_stale_limit attr value will directly limit the lru size
>> + *
>> + * @param[in] cache  a memory registration cache object
>> + * @param[in] entry  a memory registration cache entry
>> + *
>> + * @return           FI_SUCCESS, always
>> + */
>> +static inline int __mr_cache_lru_push(
>push/pop typically refers to a stack implementation. This would be a
>queue.
>—
>Reply to this email directly or
>view it on GitHub 
><https://github.com/ofi-cray/libfabric-cray/pull/202/files#r31541608>.
>
>